include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -265,6 +267,9 @@ MatrixFree<dim, Number, VectorizedArrayType>::internal_reinit(
   const typename MatrixFree<dim, Number, VectorizedArrayType>::AdditionalData
     &additional_data)
 {
+  // Store the level of the mesh to be worked on.
+  this->level_mg_handler = additional_data.level_mg_handler;
```

Comment:
This suggests that the name is already awkward in the `AdditionalData` structure.

Reply:
Indeed it is - the reason for the name is that, back in 2011 when this code was written, we had a separate `MGDoFHandler` object, so the name was at least tangentially relevant because it referred to the case of a level when we had an `MGDoFHandler` type. But yes, it is a bit confusion; we could select a new one and deprecate this one, but as all interface breaks this will incur some noise for the applications.

Reply:
@kronbichler I do not think changing the name would be a problem. We create some form of alias and deprecate this... But do you agree that the name of this variable and the method should be called consistently. If yes, what name do you prefer?